yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -149,17 +156,19 @@ func setImportTableListFlag() {
 }
 
 func parseFileTableMapping() {
+	ds = datastore.NewDataStore(dataDir)
```

Comment:
This doesn't look like a right place for this.
Initialise `ds` as a first statement in the `Run()` method of both `import data` and `import data file`. So, the rest of the code paths can always assume that `ds` is already initialised.